gensim/test/test_utils.py

```diff
@@ -84,6 +85,48 @@ def test_decode_entities(self):
         expected = u'It\x92s the Year of the Horse. YES VIN DIESEL \U0001f64c \U0001f4af'
         self.assertEqual(utils.decode_htmlentities(body), expected)
 
+    def test_open_file_existent_file(self):
+        exc = False
+        try:
+            with utils.open_file(datapath('testcorpus.mm')):
+                pass
+        except Exception:
+            exc = True
+
+        self.assertFalse(exc)
+
+    def test_open_file_non_existent_file(self):
```

Comment:
Please use something like Please use something like https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises (instead of manual wrapping to try/exc )